### PR TITLE
Update school description and logo

### PR DESCRIPTION
**Pull request by pensierinmusica**

### CODE REVIEW

**Comment by pensierinmusica** [comment]
@tati @cecyc @sethbaur @kkleinhans any update on this PR?

Thanks!

File: bootcamps/codeworks/data.yml [hunk 1]
```diff
@@ -1,45 +1,31 @@
...
     admissions: Application, interviews, coding assignment
     cities:
-    - barcelona
+    - Austin
+    - Barcelona
+    - Berlin
+    - London
+    - New York
+    - Toronto
+    - Remote
```
Comment (cecyc) [inline bootcamps/codeworks/data.yml]: these should be lower cased

File: bootcamps/codeworks/data.yml [hunk 2]
```diff
@@ -1,45 +1,31 @@
...
     duration: 12
     duration_units: weeks
     financing: true
-    guarantee: 'No'
+    guarantee: Yes - within one year of graduation or your money back
     outcomes_verified: true
     placement: Undisclosed
-    reports_outcomes: true
-    scholarships: For women and when joining the course with friends
+    reports_outcomes: https://codeworks.me/outcomes/
```
Comment (cecyc) [inline bootcamps/codeworks/data.yml]: should be a boolean (true or false)

**Comment by pensierinmusica** [comment]
@cecyc thanks! Is this correct now?

**Comment by cecyc** [comment]
@pensierinmusica just one more thing: `new york` in lines 14 and 61 needs to be hyphenated. Here's an example from another file: https://github.com/Thinkful/bootcamp-finder/blob/26649458ebb10fa3701c49df6b3b9ffaa34211fd/bootcamps/thinkful/data.yml#L42

Hope that helps!

**Comment by pensierinmusica** [comment]
@cecyc done! Thx for helping out  :)

**Pull request merged**